pyext/src/output.py

```diff
@@ -692,59 +692,148 @@ def write_stats2(self):
         for stat in self.dictionary_stats2.keys():
             self.write_stat2(stat)
 
-
 class ProcessOutput(object):
     """A class for reading stat files"""
     def __init__(self, filename):
         self.filename = filename
-        self.isstat1 = False
-        self.isstat2 = False
+        self.isstat1=False
+        self.isstat2=False
 
         # open the file
         if not self.filename is None:
             f = open(self.filename, "r")
         else:
             raise ValueError("No file name provided. Use -h for help")
 
-        # get the keys from the first line
-        for line in f.readlines():
-            d = ast.literal_eval(line)
-            self.klist = list(d.keys())
-            # check if it is a stat2 file
-            if "STAT2HEADER" in self.klist:
-                self.isstat2 = True
-                for k in self.klist:
-                    if "STAT2HEADER" in str(k):
-                        # if print_header: print k, d[k]
-                        del d[k]
-                stat2_dict = d
-                # get the list of keys sorted by value
-                kkeys = [k[0]
-                         for k in sorted(stat2_dict.items(), key=operator.itemgetter(1))]
-                self.klist = [k[1]
-                              for k in sorted(stat2_dict.items(), key=operator.itemgetter(1))]
-                self.invstat2_dict = {}
-                for k in kkeys:
-                    self.invstat2_dict.update({stat2_dict[k]: k})
-            else:
-                IMP.handle_use_deprecated("statfile v1 is deprecated. "
-                                          "Please convert to statfile v2.\n")
-                self.isstat1 = True
-                self.klist.sort()
+        self.exp_dict={} # Store all experimental header details here.
+
+        # get the {CategoryID:Category} pairs from the first line
+        line = f.readline()
+
+        #Store these keys in a dictionary. Example pair: {109 :'Total_Score'}
+        self.dict = self.parse_line(line, header=True)
+        #self.dict = ast.literal_eval(line)
```

Comment:
Remove, not comment out, old stuff. If we want to go back, that's what git is for.